### PR TITLE
Added explicit node and npm versions to travis config.

### PR DESCRIPTION
**Pull request by mikeller**

### CODE REVIEW

**Comment by McGiverGim** [comment]
Maybe is good to replace this info into the instructions of the `readme`, I'm talking about the `npm` installation:
> npm i -g npm@latest

**Comment by mikeller** [comment]
The way this is working now is somewhat suboptimal because the build steps are running until a time out after 2 minutes.
But it is a bunch better compared to not building at all in CI, so I think this should go in.

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
+os:
+  - linux
+  - osx
+
+node_js:
+  - node
+
 before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linuxx" ]]; then sudo apt-get -qq update; fi
```
Comment (McGiverGim) [inline .travis.yml]: This is correct? linuxx with two x? I don't know nothing about travis

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install -y rpm; fi
+  - npm i -g npm@latest
   - npm install gulp -g
 
 script:
   - npm install
   - gulp release
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then gulp release --win32; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then gulp release --chromeos; fi
```
Comment (McGiverGim) [inline .travis.yml]: And here both without the two x?

File: gulpfile.js [hunk 3]
```diff
@@ -496,8 +496,7 @@ function release_osx64() {
                     }
                 }
             },
-        })
-    );
+        }));
```
Comment (McGiverGim) [inline gulpfile.js]: Maybe is better the original indentation

**Pull request merged**